### ISSUE TITLE
Should Hubs extend activity pub?

### ISSUE THREAD

**Issue by OR13** [decentralized-identity/secure-data-store#106]
https://www.w3.org/ns/activitystreams#activitypub

I vote yes, since we are not seeing any real definition for them, and we need to start somewhere.

**Comment by OR13** [comment]
cc @csuwildcat what if hubs were built on top of a W3C standard, and worked withe encrypted data vaults?

**Comment by csuwildcat** [comment]
Activity Pub fails to deliver the most important aspects of Hubs, so I don't see how using them is a significant benefit to achieving the desired ends.

**Comment by OR13** [comment]
> Activity Pub fails to deliver the most important aspects of Hubs, so I don't see how using them is a significant benefit to achieving the desired ends.

I don't agree. I feel activity pub is better starting point for hubs than a blank slate.
